firestore/app/src/main/java/com/google/example/firestore/kotlin/DocSnippets.kt

```diff
@@ -847,6 +851,10 @@ abstract class DocSnippets(val db: FirebaseFirestore) {
         citiesRef.whereIn("country", listOf("USA", "Japan"))
         // [END in_filter]
 
+        // [START not_in_filter]
+        citiesRef.whereNotIn("country", Arrays.asList("USA", "Japan"))
```

Comment:
Kotlin offers a `listOf()` function that could make this simpler.

```suggestion
        citiesRef.whereNotIn("country", listOf("USA", "Japan"))
```